api/controllers/PersonalController.js

```diff
@@ -53,16 +57,19 @@ module.exports = {
       await MedicalReport.upsert({
         applicationCode: req.session.applicationCode,
         socialInsuranceNumber: req.body.socialInsuranceNumber,
```

Comment:
holdover from the previous design... I've gone blind to all these things that have been there for so long, didn't even notice them gone on the new designs... I'll get them all removed.

Regarding SIN validation, it will validate with or without spaces. The important part is that it passes the LUHN algorithm test.

Reply:
Awesome! We never had that hint text in Figma, just one of those things that appears. Weird!